jujugui/static/gui/src/app/jujulib/bakery.js

```diff
@@ -71,8 +71,8 @@ var module = module;
       this._client = httpClient;
       this.storage = storage;
       this._onSuccess = params.onSuccess || (() => {});
-      this._visitPage = params.visitPage || (visitURL => {
-        window.open(visitURL, 'Login');
+      this._visitPage = params.visitPage || (error => {
```

Comment:
The constructor params didn't change, but I've updated the part about visitPage to note the error object instead of the url.